packages/renderer/src/lib/kube/KubernetesDashboardResourceCard.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+import type { Component } from 'svelte';
+import { router } from 'tinro';
+
+let {
+  type,
+  Icon,
+  count,
+  link,
+}: {
+  type: string;
+  Icon: Component;
+  count: number;
+  link: string;
+} = $props();
+</script>
+
+<!-- svelte-ignore a11y_missing_attribute -->
+<!-- svelte-ignore a11y_no_static_element_interactions -->
+<!-- svelte-ignore a11y_click_events_have_key_events -->
+<a class="flex flex-col gap-4 p-4 bg-[var(--pd-invert-content-card-bg)] rounded-md" onclick={() => router.goto(link)}>
+  <div class="font-bold text-[var(--pd-invert-content-card-text)]">{type}</div>
+    <div class="grid grid-cols-2 gap-4">
+        <div><Icon size={40}/></div>
+        <div class="text-2xl text-[var(--pd-invert-content-card-text)]" aria-label="{type} count">
+        {count}
+        </div>
+    </div>
+  </a>
```

Comment:
issue: missing EOF, linter/formatter is not working ?

Reply:
Correct, as with above.